examples/pull.rs

```diff
@@ -8,12 +8,13 @@ use tmq::{pull, Context, Result};
 #[tokio::main]
 async fn main() -> Result<()> {
     if let Err(_) = env::var("RUST_LOG") {
-        env::set_var("RUST_LOG", "subscribe=DEBUG");
+        env::set_var("RUST_LOG", "pull=DEBUG");
```

Comment:
We should really change this to not use `set_var` since that's considered unsound & maybe we wanna update to `tracing`